### PR TITLE
Feature/gbi 1140 upgrade bitcoin node

### PR DESCRIPTION
**Pull request by coolwather**

### CODE REVIEW

**Review by rafaiovlabs — COMMENTED** [review]
Erase commented lines here, if commented and not needed we should erase them 
.

**Review by rafaiovlabs — CHANGES_REQUESTED** [review]
Please erase commented lines on Dockerfiles. if we need a comment please add them only if there are truly necessary.

File: docker-compose/lps/start.sh [hunk 1]
```diff
@@ -4,6 +4,23 @@ set -e
...
 
+echo "Testing if we have a default wallet"
+
+curl -s "http://bitcoind01:5555" --user "$BTCD_RPC_USER:$BTCD_RPC_PASS" -H "Content-Type: application/json" -d '{"jsonrpc": "1.0", "method": "getwalletinfo", "params": [], "id":"getwallet"}' | grep "{\"result\":null,\"error\":{\"code\":-18" \
+  && echo "No default wallet" \
+  && echo "Creating wallet" \
+  && curl -s "http://bitcoind01:5555" --user "$BTCD_RPC_USER:$BTCD_RPC_PASS" -H "Content-Type: application/json" -d '{"jsonrpc": "1.0", "method": "createwallet", "params": ["main", false, false, "", true, false, true], "id":"createwallet"}' \
+
+
+jq \
```
Comment (Luisfc68) [inline docker-compose/lps/start.sh:15]: I think this is not necessary because we're not using config.json anymore

File: it/config.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (Luisfc68) [inline it/config.json:1]: This file is no longer needed with GBI-1123

**Review by Luisfc68 — COMMENTED** [review]
LGTM

File: docker-compose/lps/start.sh (hunk 1 above)
Comment (coolwather) [reply docker-compose/lps/start.sh:15]: It's better to create a task for it and do the proper test.
Just to keep track.

File: it/config.json (hunk 2 above)
Comment (coolwather) [reply it/config.json:1]: Was added to gitignore

**Review by Luisfc68 — APPROVED** [review]

**Pull request merged**